sites/example-project/tailwind.config.cjs

```diff
@@ -9,6 +9,7 @@ module.exports = {
 			'../../pages/**/*.{html,js,svelte,ts,md}', // This is used in end user projects to let them access tailwind classes
 			'../../src/**/*.{html,js,svelte,ts,md}', // This is used in end user projects to let them access tailwind classes
 			'./node_modules/@evidence-dev/core-components/dist/**/*.{html,js,svelte,ts,md}',
+			'./node_modules/@evidence-dev/console/dist/**/*.{html,js,svelte,ts,md}',
```

Comment:
Do we want to consider something along these lines? This might make things a little less optimal for the sake of simplicity
```suggestion
			'./node_modules/@evidence-dev/**/dist/**/*.{html,js,svelte,ts,md}',
```

Reply:
I think it would be better to make it computed based on all the plugins in the `evidence.plugins.yaml` so other components plugins can use tailwind as well